Modules/SubModule_Import/xASL_bids_BIDSifyCheckTimeEncoded.m

```diff
@@ -0,0 +1,90 @@
+function [jsonIn,jsonOut,bTimeEncoded,bTimeEncodedFME] = xASL_bids_BIDSifyCheckTimeEncoded(jsonIn,jsonOut)
```

Comment:
I do not see that the jsonIn is anywhere modified. And it wouldn't even make much sense. For more transparency, please exclude jsonIn from the output parameters - everywhere in the header and also in all function calls of this function.

Reply:
Easy fix 4df68b9 👍